### DISCUSSION TITLE
Gerät lässt sich nicht aus HomeKit entfernen

### DISCUSSION THREAD

**Discussion by thomas-br** [thkl/hap-homematic#691] [Q&A] [answered]
Ich hatte ein HmIP-DLD Türschloss über hap-homematic an mein HomeKit angebunden. Nachdem ich das Gerät aus hap-homematic gelöscht, ist dieses leider in HomeKit weiterhin unter dem "Nicht erreichbar" Status aufgelistet, ohne einen Löschen-Button. Hat jemand eine Idee wie ich es aus HomeKit rausbekomme?

**Comment by jb-home** [comment, marked as answer]
Wieder einbinden und dann zuerst aus HomeKit löschen.
Ggf. geht es auch über die EVE App.
